vnext/Desktop.IntegrationTests/WebSocketIntegrationTest.cpp

```diff
@@ -32,19 +36,23 @@ TEST_CLASS(WebSocketIntegrationTest)
       connected = true;
     });
 
+    server->Start();
```

Comment:
Looks like Start and Stop calls need to be paired. Would it make sense to have an RAII wrapper for that?

Reply:
Currently I need explicit control of this life cycle because the `Stop` method makes the background thread join.

Reply:
To Andreas' point though, wouldn't get that by using RAII? It makes usage of the component easier, since you don't need to remember to call Stop when tearing down the server

Reply:
I prefer the server not to start on creation, so it can be configured (handlers added) after instantiation.

This is equivalent to the HTTP test server's implementation.